docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -150,6 +151,29 @@ Users can install and run multiple versions of the .NET Framework on their compu
 
     - It checks in order from most recent version to earliest version.
 
+<a name="ps_a"></a> 
+#### To check for a minimum-required .NET Framework version by querying the registry in PowerShell (.NET Framework 4.5 and later)
+
+- The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed (returning `True` if it is and `False` otherwise). You can substitute another value listed in the table to check for a different minimum-required .NET Framework version.
+
+    |Version|Minimum value of the Release DWORD|
+    |-------------|--------------------------------|
+    |.NET Framework 4.5|378389|
```

Comment:
Because this example checks "for a minimum-required .NET Framework version", its reference table deliberately consolidates by Framework version, omitting distinct version numbers for different operating systems.